### PR TITLE
feat: don't require `lib` directory for `Packager` operations

### PR DESCRIPTION
**Pull request by mlutze**
fixes https://github.com/flix/flix/issues/3099

### CODE REVIEW

**Comment by magnus-madsen** [comment]
So we still create dir right?

File: main/src/ca/uwaterloo/flix/tools/Packager.scala [hunk 1]
```diff
@@ -222,13 +222,16 @@ object Packager {
...
     // Add all library packages.
-    for (file <- getAllFiles(getLibraryDirectory(p))) {
-      if (file.getFileName.toString.endsWith(".fpkg")) {
-        // Case 1: It's a Flix package.
-        flix.addSourcePath(file)
-      } else if (file.getFileName.toString.endsWith(".jar")) {
-        // Case 2: It's a JAR.
-        flix.addJar(file)
+    val lib = getLibraryDirectory(p)
+    if (lib.toFile.exists()) {
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/tools/Packager.scala:226]: Files.exists and Files.isDirectory (it could exist, but be a file)

**Comment by mlutze** [comment]
> So we still create dir right?

Yep

**Comment by mlutze** [comment]
!!!jar

**Comment by mlutze** [comment]
Everything seems to be working fine both with and without a `lib` directory.

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**